qhub/deploy.py

```diff
@@ -156,31 +158,31 @@ def provision_06_kubernetes_keycloak_configuration(stage_outputs, config, check=
         ),
     )
 
-    if check:
+    if not disable_checks:
         checks.stage_06_kubernetes_keycloak_configuration(stage_outputs, config)
 
 
-def provision_07_kubernetes_services(stage_outputs, config, check=True):
+def provision_07_kubernetes_services(stage_outputs, config, disable_checks=False):
     directory = "stages/07-kubernetes-services"
 
     stage_outputs[directory] = terraform.deploy(
         directory=directory,
         input_vars=input_vars.stage_07_kubernetes_services(stage_outputs, config),
     )
 
-    if check:
+    if not disable_checks:
         checks.stage_07_kubernetes_services(stage_outputs, config)
 
 
-def provision_08_qhub_tf_extensions(stage_outputs, config, check=True):
+def provision_08_qhub_tf_extensions(stage_outputs, config, disable_checks=False):
     directory = "stages/08-qhub-tf-extensions"
 
     stage_outputs[directory] = terraform.deploy(
         directory=directory,
         input_vars=input_vars.stage_08_qhub_tf_extensions(stage_outputs, config),
     )
 
-    if check:
```

Comment:
I often find double negatives harder to reason about but don't have a strong opinion that it must be this way. What do you think?

```
provision_02_infrastructure(stage_outputs, config, disable_checks)

def provision_02_infrastructure(stage_outputs, config, disable_checks=False):
      if not disable_checks:
          ...
```

vs. 

```python
provision_02_infrastructure(stage_outputs, config, (not disable_checks))

def provision_02_infrastructure(stage_outputs, config, check=True):
      if check:
          ...
```

Reply:
Yeah I understand where you're coming from. In this case, I feel like it reads in *fairly* comprehensible way but I'm open to other options as well. I do find this pattern a little pattern a little confusing myself `provision_02_infrastructure(stage_outputs, config, (not disable_checks))`. 

Perhaps we could do something like this instead:

```
    if disable_checks:
        return 
    checks.stage_02_infrastructure(stage_outputs, config
```